shell/deepdive-run

```diff
@@ -108,6 +108,9 @@ fullConfig=$run_dir/deepdive.conf
         echo 1
     ) 2>/dev/null)}
     export PARALLELISM
+    
+    # set INPUT_BATCH_SIZE env var 
+    ! [[ -e "$INPUT_BATCH_SIZE" ]] || export INPUT_BATCH_SIZE=10000
```

Comment:
First, when `INPUT_BATCH_SIZE` is not defined, the entire script aborts.  Second, this test with `-e` is checking whether there *exists* a file named with that value which doesn't make sense.

Here's the right way to do this:
```bash
: ${INPUT_BATCH_SIZE:=10000}
export INPUT_BATCH_SIZE
```